groups/instance/cloud-init/templates/artifactory.yml.tpl

```diff
@@ -0,0 +1,21 @@
+#cloud-config
+fqdn: ${instance_fqdn}
+runcmd:
+  - wget https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-j-8.0.31.zip
+  - yum install unzip -y
+  - yum install mariadb -y
+  - unzip mysql-connector-j-8.0.31.zip
+  - cp mysql-connector-j-8.0.31/mysql-connector-j-8.0.31.jar /opt/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib
+  - mysql --version
```

Comment:
Just checking if this connector is required? If so the install should live within the AMI build.